sci-astronomy/stellarium/stellarium-0.22.1.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit cmake desktop flag-o-matic python-any-r1 xdg virtualx
+
+DESCRIPTION="3D photo-realistic skies in real time"
+HOMEPAGE="https://stellarium.org/"
+MY_DSO_VERSION="3.13"
```

Comment:
Have you tried building it (and running) with USE=deepsky?

Stellarium 0.22.1 uses DSO 3.15: https://github.com/Stellarium/stellarium-data/releases/tag/dso-3.15

And it warns about the wrong DSO version during startup

Reply:
I did not, but I'll give it a go now and report back.

Reply:
Good catch; it was throwing a warning. Simple fix in the ebuild though!